app/features/DataMIDI.py

```diff
@@ -1,11 +1,11 @@
 from mido import MidiFile, tempo2bpm, tick2second
-import FeaturesMIDI
+from .FeaturesMIDI import FeaturesMIDI
```

Comment:
Not fan of relative imports, they introduce problems when the script is run from a different directory
Please use `app.FeaturesMIDI`